Gruntfile.cjs

```diff
@@ -25,6 +25,7 @@ module.exports = function(grunt) {
             cwd: './',
```

Comment:
When I added `type=module` to `package.json` when I tried to build the project via grunt I was getting errors which suggested I should change the grunt file extension to `cjs` when I tried that the errors went away and it seemed to build fine. _Just explaining why it's changed._